webpack/assets/javascripts/react_app/components/SettingsTable/components/__tests__/SettingCell.test.js

```diff
@@ -6,19 +6,71 @@ import {
   withoutFullName,
 } from '../../../SettingRecords/__tests__/SettingRecords.fixtures';
 
+import React from 'react';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
```

Comment:
If you use the `rtlHelper`, you don't need to use the mock store. You can use `rederWithStore` directly.